### PR TITLE
fix: wait after setting the refresh token

### PR DESCRIPTION
**Pull request by pmuir**
This is a nasty way to handle this, but we will remove this code soon anyway.

### CODE REVIEW

File: src/app/utils/keycloak/keycloakUtils.ts [hunk 1]
```diff
@@ -41,6 +46,8 @@ export const initKeycloak = async (
       await rk.init(initOptions);
       // Set the saved refresh token into Keycloak
       rk.refreshToken = refreshToken;
+      // Hack to ensure that the refresh token is properly set on the objet
```
Comment (wtrocki) [inline src/app/utils/keycloak/keycloakUtils.ts:49]:
```suggestion
      // Hack to ensure that the refresh token is properly set on the object
```

**Comment by wtrocki** [comment]
Testing on Brave is enough?

**Comment by pmuir** [comment]
Brave and Firefox with private mode are the reported browsers with issues.

**Comment by wtrocki** [comment]
It think it works but I'm not 100% sure if 100ms will not be hit

**Comment by wtrocki** [comment]
I works on Brave and Firefox- My machine is quite fast and due to resolution depending on processing I would recommend to test it on the machine that is slower to see if 100ms will be enough.

> EDIT 

Tried on firefox mobile in private mode - it works

**Comment by pmuir** [comment]
I think it's the other way around - the faster the machine, the worse the problem. If the machine is slow, the execution of the `updateToken` call will be later than on a fast machine, thus meaning the value has time to be set. This is, I think, why it shows on Brave and not Chrome (Brave is lighter weight but same engine) and Firefox Private mode (less junk running) and why you can't reproduce this in a debugger.

**Comment by wtrocki** [comment]
Approving

**Review by wtrocki — APPROVED** [review]

**Pull request merged**